Documentation/op-guide/security.md

```diff
@@ -2,7 +2,7 @@
 title: Transport security model
 ---
 
-etcd supports automatic TLS as well as authentication through client certificates for both clients to server as well as peer (server to server / cluster) communication.
+etcd supports automatic TLS as well as authentication through client certificates for both clients to server as well as peer (server to server / cluster) communication. **Note that etcd doesn't enable [RBAC based authentication][auth] or the authentication feature in the transport layer by default to reduce friction for users getting started with the database. Further, changing this default would be a breaking change for the project which was established 7 years ago. An etcd cluster which doesn't enable the security features can expose its data to any clients. Users need to enable the features for avoiding such a situation.**
```

Comment:
s%the security features%security features%g

Personally I would just remove “ Users need to enable the features for avoiding such a situation” completely. I don’t think it adds anything to the warning.

Reply:
Thanks for your comment. I updated based on your comment in the latest version, could you take a look again?

Reply:
@mitake @philips I would also remove mentioning number of years. So I guess we can remove `which was established 7 years ago` altogether if that sounds good? Thanks!

Reply:
good point, just say since 2013 or soemthing.

Reply:
@spzala @philips yeah that makes sense, I updated that part in the latest commit. Thanks!